src/main/java/com/google/cloud/discotoproto3converter/proto3/ProtoFile.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.discotoproto3converter.proto3;
+
+public class ProtoFile {
+  private static final String LICENSE =
+      "// Copyright 2020 Google LLC\n"
+          + "//\n"
+          + "// Licensed under the Apache License, Version 2.0 (the \"License\");\n"
```

Comment:
Can you use a `String.join("\n", ...)`?

(apparently, in Java 12, we'll be able to use raw string literals with backticks [[ref](https://www.vojtechruzicka.com/raw-strings/#raw-string-literals-in-java)])

Reply:
Using '+' is very idiomatic for java and technically more efficient (not really important here, but why do it inefficiently if we can do otherwise).